extensions/amp-script/0.1/amp-script.js

```diff
@@ -131,7 +119,10 @@ export class AmpScript extends AMP.BaseElement {
       onReceiveMessage: data => {
         dev().info(TAG, 'From worker:', data);
       },
-    }).then(workerDom => {
+      sanitizer: new SanitizerImpl(this.win),
```

Comment:
nit: bring this up with other properties (above callbacks).

Reply:
Done.